dipy/direction/peaks.py

```diff
@@ -552,15 +551,6 @@ def peaks_from_model(model, data, sphere, relative_peak_threshold,
                            odf_array if return_odf else None)
 
 
-def gfa(samples):
-    """The general fractional anisotropy of a function evaluated
```

Comment:
it might be good to warn users / devs that the function moved in another module.